YoutubeExplode/Models/MediaStreams/Extensions.cs

```diff
@@ -25,55 +29,131 @@ public static string GetFileExtension(this Container container)
+                        $"Unexpected container type [{container}].");
             }
         }
 
         /// <summary>
-        /// Gets label for given video quality and framerate
+        /// Gets label for given video quality and framerate, as displayed on YouTube.
         /// </summary>
         public static string GetVideoQualityLabel(this VideoQuality videoQuality, int framerate = 30)
         {
             // Video quality
             string qualityPart;
-            if (videoQuality == VideoQuality.Low144)
-                qualityPart = "144p";
+            switch (videoQuality)
+            {
+                case VideoQuality.Low144:
+                    qualityPart = "144p";
+                    break;
+                case VideoQuality.Low240:
+                    qualityPart = "240p";
+                    break;
+                case VideoQuality.Medium360:
+                    qualityPart = "360p";
+                    break;
+                case VideoQuality.Medium480:
+                    qualityPart = "480p";
+                    break;
+                case VideoQuality.High720:
+                    qualityPart = "720p";
+                    break;
+                case VideoQuality.High1080:
+                    qualityPart = "1080p";
+                    break;
+                case VideoQuality.High1440:
+                    qualityPart = "1440p";
+                    break;
+                case VideoQuality.High2160:
+                    qualityPart = "2160p";
+                    break;
+                case VideoQuality.High3072:
+                    qualityPart = "3072p";
+                    break;
+                case VideoQuality.High4320:
+                    qualityPart = "4320p";
+                    break;
+                default:
+                    throw new ArgumentOutOfRangeException(nameof(videoQuality),
+                        $"Unexpected video quality [{videoQuality}].");
+            }
 
-            else if (videoQuality == VideoQuality.Low240)
-                qualityPart = "240p";
+            // Framerate
+            var frameratePart = framerate > 30 ? framerate.ToString() : string.Empty;
 
-            else if (videoQuality == VideoQuality.Medium360)
-                qualityPart = "360p";
+            return qualityPart + frameratePart;
+        }
 
-            else if (videoQuality == VideoQuality.Medium480)
-                qualityPart = "480p";
+        /// <summary>
+        /// Gets all available <see cref="MediaStreamInfo"/> in a <see cref="MediaStreamInfoSet"/>.
+        /// </summary>
+        public static IEnumerable<MediaStreamInfo> GetAll(this MediaStreamInfoSet streamInfoSet)
+        {
+            streamInfoSet.GuardNotNull(nameof(streamInfoSet));
 
-            else if (videoQuality == VideoQuality.High720)
-                qualityPart = "720p";
+            foreach (var streamInfo in streamInfoSet.Muxed)
```

Comment:
`Union` also does equality comparison, the equivalent for it is actually `Concat`. I will change it so it's more readable.

Reply:
Also Muxed is short for Multiplexed

Reply:
Actually I'm not sure which one has better readability.
Do you think this is better?
```c#
return streamInfoSet.Muxed.Cast<MediaStreamInfo>().Concat(streamInfoSet.Audio).Concat(streamInfoSet.Video);
```

Reply:
Hard to tell really, I just personally don't like to reinvent the well so I use what's  already there, so I suggested you to use this.

Maybe this would be more readable if you split these into new lines?
```csharp
return streamInfoSet.Muxed.Cast<MediaStreamInfo>()
  .Concat(streamInfoSet.Audio)
  .Concat(streamInfoSet.Video);
```
The only thing I don't like is that C# compiler can't figure out common interface/base class they all use and you need to add `Cast<T>()`

Reply:
The last one looks a lot more cleaner. I'm still not convinced this is better than foreach/yield because the latter is more clear and has fewer allocations/method calls.

Reply:
Then let's go with foreach :)